packages/foundation/config/tests.js

```diff
@@ -0,0 +1,18 @@
+/** Jest test setup file. */
+
+const { setIconOptions } = require('office-ui-fabric-react/lib/Styling');
```

Comment:
can't have office-ui-fabric-react references inside foundation i believe.

Reply:
This is a test file so I didn't think it would cause issues. I copied a package as recommended but can pull it out.